dev/release/README.md

```diff
@@ -172,6 +172,74 @@ $ GH_TOKEN=${YOUR_GITHUB_TOKEN} dev/release/bump_version.sh 19.0.0-SNAPSHOT
 It creates a feature branch and adds a commit that bumps version. This
 opens a pull request from the feature branch by `gh pr create`.
 
+### Close the GitHub milestone
+
+Close the milestone here, then open a new milestone for the next release:
```

Comment:
What is the general rule for setting the target date of the next milestone?

Reply:
I'm favor of predictable release pace but flexible. 
For the community, it's great to know in advance when a release is planned. However, we should clearly state that what is in the release could change (depending of resources, priority, etc).

Reply:
I'm happy to volunteer for arrow-Java release every 6-8 weeks.

Reply:
Maybe we can trade off to reduce the burden/improve bus factor, but that sounds good to me.

Do you want to start a small discussion on the ML? Or if we're in agreement here then let's just go with it.

Reply:
Sure. I think it makes sense to discuss on the ml. Let me start a thread. Thanks !

Reply:
Releasing arrow-java every 6-8 weeks sounds a little bit frequent to me. Perhaps we can start with a three-month release cadence same to the mono repo in the past.

Reply:
Yes, let's start with quarter releases. We can always do intermediate release if needed.